### ISSUE TITLE
A11y: Back to Top has no discernable text at mobile breakpoint

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#735]
The Back to Top link is being flagged because the text disappears at mobile breakpoints.  Move the text to an aria-label or title to prevent flagging.

*NOTE - Wave did not pick this up because it does not work through mobile breakpoints

> **ESTIMATE** 1
